src/arch/linux/mapping/mapping.c

```diff
@@ -42,9 +42,9 @@
 #endif
 
 #ifdef __i386__
-enum { MEM_BASE, VM86_BASE, MAX_BASES };
+enum { MEM_BASE, VM86_BASE, KVM_VGA_BASE, MAX_BASES };
```

Comment:
Does it help adding it here?
If it does then fine, but AFACS this is
more a special case than a new mem_base.
If in the future we want a memcpy vga,
then this KVM_VGA_BASE will alias the
vga mem, but on any real mem_base
there would be unaliased space that we
later memcmp to this one.
So will it help to not create a new mem_base
from this?

Reply:
For example you can cover that
range in dosaddr_to_unixaddr()
and in kvm.c use that for
MAPPING_VGAEMU (like "directly
from source" technique).

Reply:
You actually do not even refer to
that base via MEM_BASE32x().
It seems to be used only in a few
cludges under `if (is_kvm_map())`